addons/pos_event/models/pos_order_line.py

```diff
@@ -0,0 +1,21 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields, api
+
+
+class PosOrderLine(models.Model):
+    _inherit = 'pos.order.line'
+
+    event_id = fields.Many2one('event.event', string='Event', compute="_compute_event_id", store=True, precompute=True)
+    event_ticket_id = fields.Many2one('event.event.ticket', string='Event Ticket')
```

Comment:
Unless you need to set it, could probably be compute based on event registrations? Assuming all registrations are known to have the same ticket type. As I assume there's a decent number of rows on this table could be good to keep columns low.

Reply:
We need to set it to the order in the pos frontend, when saving the order, it will create the registration.